src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfigTest.java

```diff
@@ -1,15 +1,22 @@
 package io.jenkins.plugins.artifact_manager_jclouds.s3;
 
-import hudson.model.Failure;
-import hudson.util.FormValidation;
-import io.jenkins.plugins.artifact_manager_jclouds.BlobStoreProvider;
-import io.jenkins.plugins.artifact_manager_jclouds.JCloudsArtifactManagerFactory;
 import java.util.logging.Logger;
-import jenkins.model.ArtifactManagerConfiguration;
-import static org.junit.Assert.*;
+
 import org.junit.Rule;
 import org.junit.Test;
 import org.jvnet.hudson.test.JenkinsRule;
+import io.jenkins.plugins.artifact_manager_jclouds.BlobStoreProvider;
+import io.jenkins.plugins.artifact_manager_jclouds.JCloudsArtifactManagerFactory;
+
+import hudson.model.Failure;
```

Comment:
Unfortunate. Guess I need to set up Spotless to enforce simple lexicographical `import` sorting.